contracts/VestingEscrowFactory.vy

```diff
@@ -61,12 +68,19 @@ def deploy_vesting_contract(
     @param amount Amount of tokens being vested for `recipient`
     @param vesting_duration Time period over which tokens are released
     @param vesting_start Epoch time when tokens begin to vest
+    @param cliff_length Duration after which the first portion vests
+    @param escrow_type Escrow type to deploy 0 - `VestingEscrowSimple`, 1 - `VestingEscrowFullyRevokable`
     """
-    assert cliff_length <= vesting_duration  # dev: incorrect vesting cliff
-    escrow: address = create_forwarder_to(self.target)
-    assert ERC20(token).transferFrom(msg.sender, self, amount)  # dev: funding failed
-    assert ERC20(token).approve(escrow, amount)  # dev: approve failed
-    VestingEscrowSimple(escrow).initialize(
+    assert cliff_length <= vesting_duration, "incorrect vesting cliff"
+    assert escrow_type in [0,1], "incorrect escrow type"
+    escrow: address = ZERO_ADDRESS
```

Comment:
UserWarning: ZERO_ADDRESS is deprecated. Please use `empty(address)` instead.

Reply:
Done